src/test/kotlin/com/github/julianhille/sqldelight/jscipherdriver/JsDriverTest.kt

```diff
@@ -0,0 +1,231 @@
+package com.github.julianhille.sqldelight.drivers.jscipher
+
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.db.SqlPreparedStatement
+import com.squareup.sqldelight.db.use
+
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertNull
+import kotlin.test.assertTrue
+
+open class JsDriverTest {
+
+  internal lateinit var driver: SqlDriver
+
+  internal val schema = object : SqlDriver.Schema {
+    override val version: Int = 1
+
+    override fun create(driver: SqlDriver) {
+      driver.execute(
+        0,
+        """
+              |CREATE TABLE test (
+              |  id INTEGER PRIMARY KEY,
+              |  value TEXT
+              |);
+            """.trimMargin(),
+        0
+      )
+      driver.execute(
+        1,
+        """
+              |CREATE TABLE nullability_test (
+              |  id INTEGER PRIMARY KEY,
+              |  integer_value INTEGER,
+              |  text_value TEXT,
+              |  blob_value BLOB,
+              |  real_value REAL
+              |);
+            """.trimMargin(),
+        0
+      )
+    }
+
+    override fun migrate(
+      driver: SqlDriver,
+      oldVersion: Int,
+      newVersion: Int
+    ) {
+      // No-op.
+    }
+  }
+
+  @BeforeTest
+  open fun setup() {
+    val config = MemoryDatabaseConfiguration(schema, {
+      schema.create(it)
+    }, { driver, oldVersion, newVersion ->
+      schema.migrate(driver, oldVersion, newVersion)
+    })
+    driver = SqlJsCipherDriver(config)
+  }
+
+  @AfterTest
+  fun tearDown() {
+    driver.close()
+  }
+
+  @Test fun insert_can_run_multiple_times() {
```

Comment:
```suggestion
  @Test 
  fun insert_can_run_multiple_times() {
```

Reply:
Tests are just a copy from jssql i think we should stick to the code style from there for now, so we can "just" merge updated or added tests later on. Will consider adding an issue so we might update the linting later.